dojo/tools/jfrog_xray_unified/parser.py

```diff
@@ -0,0 +1,118 @@
+import json
+
+from dojo.models import Finding
+
+
+class JFrogXrayUnifiedParser(object):
+    """JFrog Xray JSON reports"""
+
+    def get_scan_types(self):
+        return ["JFrog Xray Unified Scan"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return scan_type
+
+    def get_description_for_scan_types(self, scan_type):
+        return "Import Xray Unified (i.e. Xray version 3+) findings in JSON format."
+
+    def get_findings(self, json_output, test):
+        tree = json.load(json_output)
+        return self.get_items(tree, test)
+
+    def get_items(self, tree, test):
+        items = {}
+        if 'rows' in tree:
+            vulnerabilityTree = tree['rows']
+
+            for node in vulnerabilityTree:
+                item = get_item(node, test)
+
+                # Xray has a bug with duplicate results, so save to a dictionary so we only get
+                # one finding per issue_id
+                unique_key = node['issue_id']
+                items[unique_key] = item
+
+        return list(items.values())
+
+def get_item(vulnerability, test):
+    # Some items have multiple CVEs for some reason, so get the CVE with the highest CVSSv3 score.
+    # Note: the xray v2 importer just took the first CVE in the list, that doesn't seem ideal though
+    highestCvssV3Index = 0
+    highestCvssV3Score = 0
+
+    for thisCveIndex in range(0, len(vulnerability['cves']) - 1):
+        # not all cves have cvssv3 scores, so skip these. If no v3 scores, we'll default to index 0
+        if 'cvss_v3_score' in vulnerability['cves'][thisCveIndex]:
+            thisCvssV3Score = vulnerability['cves'][thisCveIndex]['cvss_v3_score']
+            if thisCvssV3Score > highestCvssV3Score:
+                highestCvssV3Index = thisCveIndex
+                highestCvssV3Score = thisCvssV3Score
+
+    # Following the CVSS Scoring per https://nvd.nist.gov/vuln-metrics/cvss
+    if 'severity' in vulnerability:
+        if vulnerability['severity'] == 'Unknown':
+            severity = "Info"
+        else:
+            severity = vulnerability['severity'].title()
+    # TODO: Needs UNKNOWN new status in the model.
+    else:
+        severity = "Info"
+
+    cveIndex = highestCvssV3Index
+
+    cve = "No CVE on file"
+    cvss_v3 = "No CVSS v3 score."
+    cvss_v2 = "No CVSS v2 score."
+    mitigation = "N/A"
+    extra_desc = ""
+
+    cves = vulnerability.get('cves', [])
+    if len(cves) > 0:
+        worstCve = cves[cveIndex]
+        if 'cve' in cves[cveIndex]:
+            cve = worstCve['cve']
+        if 'cvss_v3_vector' in worstCve:
+            cvss_v3 = worstCve['cvss_v3_vector']
+        if 'cvss_v2_vector' in worstCve:
+            cvss_v2 = worstCve['cvss_v2_vector']
+
+    if 'fixed_versions' in vulnerability and len(vulnerability['fixed_versions']) > 0:
+        mitigation = "Versions containing a fix:\n"
+        mitigation = mitigation + "</br>".join(vulnerability['fixed_versions'])
```

Comment:
```suggestion
        mitigation = "Versions containing a fix:\n"
        mitigation = mitigation + "</br>".join(vulnerability['fixed_versions'])
```
The syntax should be Markdown here not HTML.

Reply:
Ok I'll change it to use \n instead of </br> then? I'm assuming the suggested change is a mistake since i can't see a difference in the code?

Reply:
fixed in https://github.com/DefectDojo/django-DefectDojo/pull/4551/commits/33ca0146564f1eccd28c362ff818b24c985d008a